assets/rss/mailchimp-feed-rss2.php

```diff
@@ -0,0 +1,116 @@
+<?php
```

Comment:
@mcaskill I'll let you tweak this RSS template. Right now it should output the excerpt and the featured image of a post.